vizro-core/src/vizro/models/types.py

```diff
@@ -245,6 +247,19 @@ def _check_type(cls, captured_callable: CapturedCallable, field: ModelField) ->
 
         return captured_callable
 
+    def __repr__(self):
+        args = ", ".join(f"{key}={value!r}" for key, value in self._arguments.items())
+        x = f"{self._function.__module__}.{self._function.__name__}({args})"
+        return x
```

Comment:
These are still the quick and dirty versions and should be tidied:

```suggestion
    def __repr__(self):
        args = ", ".join(f"{key}={value!r}" for key, value in self._arguments.items())
        return f"{self._function.__module__}.{self._function.__name__}({args})"
```